### PR TITLE
Schema: Document @relation cases

### PR DESCRIPTION
**Pull request by janpio**
In https://github.com/prisma/specs/pull/201 @tomhoule and @mavilein added information about the logic for ambiguous relations.

This left the resulting `@relation` section a bit too specific about this one edge case, vs. the more probable case of a named relation. This PR works around this by adding a bit more structure.

### CODE REVIEW

File: schema/Readme.md [hunk 1]
```diff
@@ -690,7 +692,13 @@ Specifies a default value if null is provided
 
 Disambiguates relationships when needed.
 
-There can be multiple distinct relationships between two models, or between a model and itself ("self relation"). When this is the case, the relationships must be named, so they can be distinguished. Relation fields that do not clearly belong to a specific relationship constitute an *ambiguous relation*.
+###### Named relations
+
+There can be multiple distinct relationships between two models, or between a model and itself ("self relation"). When this is the case, the relationships must be named, so they can be distinguished.
```
Comment (tomhoule) [inline schema/Readme.md]:
```suggestion
When a model contains a single relation to another model or itself, giving a name to the relation is optional and the `@relation directive` can be completely omitted.

There can be multiple distinct relationships between two models, or between a model and itself ("self relation"). When this is the case, the relationships must be named, so they can be distinguished. 
```

File: schema/Readme.md [hunk 2]
```diff
@@ -690,7 +692,13 @@ Specifies a default value if null is provided
 
 Disambiguates relationships when needed.
```
Comment (tomhoule) [inline schema/Readme.md]:
```suggestion
Disambiguates relationships when needed.

When a model contains a single relation to another model or itself, giving a name to the relation is optional and the `@relation directive` can be completely omitted.
```

File: schema/Readme.md (hunk 1 above)
Comment (tomhoule) [inline schema/Readme.md]:
```suggestion
When a model contains a single relation to another model or itself, giving a name to the relation is optional and the `@relation directive` can be completely omitted.

There can be multiple distinct relationships between two models, or between a model and itself ("self relation"). When this is the case, the relationships must be named, so they can be distinguished. 
```

**Comment by janpio** [comment]
Assuming silent agreement after 7 days.

**Pull request merged**